app/javascript/components/upload/H5adFileForm.jsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react'
```

Comment:
Blocker: As part of substantive architecture refactor for h5ad -> AnnData: 

Let's rename this file to e.g. `AnnDataFileForm`.